sdcm/test_config.py

```diff
@@ -193,9 +191,22 @@ def configure_ldap(cls, node, use_ssl=False):
         if ContainerManager.get_container(node, 'ldap').exec_run("timeout 30s container/tool/wait-process")[0] != 0:
             raise LdapServerNotReady("LDAP server didn't finish its startup yet...")
 
+    @classmethod
+    def _link_running_syslog_logdir(cls, syslog_logdir):
+        current_logdir = cls.logdir()
+        if not syslog_logdir or current_logdir == syslog_logdir:
```

Comment:
assert when `not syslog_logdir`

Reply:
use proper warning message when `current_logdir == syslog_logdir`

Reply:
fixed